server/controllers/importers/graphml.js

```diff
@@ -2,7 +2,7 @@ var constants = require(__dirname + "/../constants");
 var parseString = require("xml2js").parseString;
 var semanticChecker = require(__dirname + "/../semantic-checker");
 
-/* 
+/*
```

Comment:
It looks like Eddie has a few blocks of code commented out here so that he can return to where he left off on GraphML syntactic tests after the summer.